app/views/assignments/show.html.erb

```diff
@@ -21,7 +21,7 @@
         <div class="assignments-data">
           <p><%= t("assignments.assignment_data.name_html", name: @assignment.name) %></p>
           <p><%= t("assignments.assignment_data.group_html", group_name: @assignment.group.name) %></p>
-          <p><strong><%= t("assignments.deadline") %></strong><span class="assignments-deadline" data-year=<%= deadline_year(@assignment) %>
+          <p><strong><%= t("assignments.deadline") %></strong> <span class="assignments-deadline" data-year=<%= deadline_year(@assignment) %>
```

Comment:
Sorry my bad, could you try adding space before the `/strong ` tag.
If that still gives error then this can be fixed by adding space in the translations-
[here](https://github.com/CircuitVerse/CircuitVerse/blob/a92acd5f460092ee4a803b857745e4c2793b62c3/config/locales/views/assignments/en.yml#L4)